### PR TITLE
Need ncurses for pretty printing to work

### PR DESCRIPTION
**Pull request by dlorch**
- [x] I have reviewed the [Contributor Guidelines][contributor].
- [x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

As the title says. Without ncurses, the output defaults to "TAP", even on an interactive console:

```
$ docker run -it bats/bats:latest /opt/bats/test
1..78
ok 1 no arguments prints message and usage instructions
ok 2 invalid option prints message and usage instructions
ok 3 -v and --version print version number
ok 4 -h and --help print help
ok 5 invalid filename prints an error
ok 6 empty test file runs zero tests
ok 7 one passing test
ok 8 summary passing tests
...
```

with ncurses:

```
$ docker run -it bats/bats:latest /opt/bats/test
 ✓ no arguments prints message and usage instructions
 ✓ invalid option prints message and usage instructions
 ✓ -v and --version print version number
 ✓ -h and --help print help
 ✓ invalid filename prints an error
 ✓ empty test file runs zero tests
 ✓ one passing test
 ✓ summary passing tests
...
```

Note that -p and --pretty do not work without ncurses.

I don't know if ncurses is actually needed or not, but bats checks for it [here]:

`command -v tput`

[here]: https://github.com/dlorch/bats-core/blob/master/libexec/bats-core/bats#L94

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ FROM bash:${bashver}
 # context where it make sense to cite GNU Parallel).
 RUN apk add --no-cache parallel && \
```
Comment (sublimino) [inline Dockerfile]: Please add `ncurses` after `parallel` so it's all in the same Docker image layer and remove the separate line 🙏

Comment (dlorch) [reply Dockerfile]: Fixed in last commit.

**Comment by sublimino** [comment]
Thanks for the contribution @dlorch! :+1:

**Comment by sublimino** [comment]
It occurs to me now this is merged that this is redefining accidental behaviour that CI suites may have been relying on: implicit default in Docker was `--tap`, it's not `--pretty`

If this will cause anybody problems please LMK here.

**Pull request merged**